pkg/models/comment.go

```diff
@@ -30,6 +30,7 @@ type Comment struct {
 	CoverTitle          string
 	CoverContent        string
 	CoverReason         string
+	Reward			    int  `gorm:"default:0"`
```

Comment:
No need to set default value here, it's already the default.

```suggestion
	Reward			    int
```